site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -35,6 +35,9 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 ==== Request Body
 
 [.api]
+[field]#displayName# [type]#[String]# [optional]#Optional#::
```

Comment:
Do we have a `since` value? Was this added after links were added? I think so.

Reply:
This was part of the original design, I believe.  This field was removed in error.